code/modules/mob/living/simple_animal/abnormality/he/woodsman.dm

```diff
@@ -251,7 +251,7 @@
 
 /mob/living/simple_animal/hostile/abnormality/woodsman/zero_qliphoth(mob/living/carbon/human/user)
```

Comment:
I should point out that this doesn't work because zero_qliphoth only gets called once.
If abnormality is already at 0 it won't be called.

Reply:
Thank you for that information